modules/oe_translation_cdt/src/Model/Transaction.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_translation_cdt\Model;
+
+use Symfony\Component\Serializer\Attribute\SerializedPath;
+
+/**
+ * Defines the DTO class for transaction.
+ */
+class Transaction {
+
+  #[SerializedPath('[TransactionHeader][SenderDetails][@drupalVersion]')]
```

Comment:
In the current version of `drupal/coder`, this must go first, before the actual comment. The fix is on the way (https://www.drupal.org/project/coder/issues/3322615).

I used PHP attributes instead of annotations, because serializer annotations are deprecated and will be removed in SF7/D11.